File: app/src/main/java/com/oxodiceproductions/dockmaker/Database/Document.java
package com.oxodiceproductions.dockmaker.Database;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Document {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    long id;

    @ColumnInfo(name = "time")
    long time;

    @ColumnInfo(name = "name")
    String name;

    public Document(long time, String name) {
        this.time = time;
        this.name = name;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getTime() {
        return time;
    }

    public void setTime(long time) {
        this.time = time;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/oxodiceproductions/dockmaker/database/Document.kt
package com.oxodiceproductions.dockmaker.database

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class Document(
    @field:ColumnInfo(name = "time") var time: Long, @field:ColumnInfo(
        name = "name"
    ) var name: String
) {
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    var id: Long = 0

}